### PR TITLE
added standart english layout with german umlauts

### PR DESCRIPTION
**Pull request by philipp8101**
hey, just discovered this app, seems like a really cool alternative to regular keyboards. (now i just have to get gud on it 😂)
i started out on the English layout, and rather stick with it, so i added the German symbols i need.
i tried it with the debug APK in the emulator, but i cant seem to find the new layout in the selection 😕
many thanks, Philipp

### CODE REVIEW

**Comment by dessalines** [comment]
There is already a ENDEThumbKey keyboard, did you try that out?

**Comment by philipp8101** [comment]
Have you considered, making the layouts simple configuration files, that can be loaded in at runtime.
Rather than compiling them in, and releasing a new version for every new layout.

**Comment by dessalines** [comment]
It's called english deutsch bilingual thumb-key.

**Comment by dessalines** [comment]
Also what version are you on?

**Comment by philipp8101** [comment]
I'm on 2.6.9 that's the latest on f-droid.

**Comment by dessalines** [comment]
We're way past that, use either izzyondroid or github to get the newest version.

**Comment by philipp8101** [comment]
I'm on 2.7.2 now, and have tried the layout you suggested. What I want is the same layout as "english thumb-key" but with the extra german letters.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ENUmlautThumbKey.kt [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+            listOf(
+                SPACEBAR_KEY_ITEM,
+                RETURN_KEY_ITEM,
+            ),
+        ),
+    )
+
+val KB_EN_WITH_UMLAUT_THUMBKEY: KeyboardDefinition =
+    KeyboardDefinition(
+        title = "english thumb-key with german umlauts",
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ENUmlautThumbKey.kt:565]: Alright, in that case can you rename this like the other keyboard, but v2? Something like `english deutsch thumb-key v2`

**Pull request merged**